code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -29,33 +29,33 @@
 
 /mob/living/simple_animal/cat/Life()
 	..()
+	if(!stasis)
```

Comment:
Just do if(stasis) return, rather than tabbing everything

Reply:
That makes cat autistic. Or I misunderstood what you wanted, either way I leave thing as it is for now.